sync/syncer_test.go

```diff
@@ -344,7 +336,7 @@ func Test_TwoNodes_SyncIntegrationSuite(t *testing.T) {
 	i := 1
 	sis.BeforeHook = func(idx int, s p2p.NodeTestInstance) {
 		l := log.New(fmt.Sprintf("%s_%d", sis.name, i), "", "")
```

Comment:
the second and third args of the new methods choose in which file this log will be saved. left empty the log isn't save to file.
The best practice (in my opinion) will be to create the log at the most top level and pass it down to each component so all the logs can be centered in one file.

Reply:
currently log is initialized in the p2p layer so you wouldn't be able to do that. I'll open an issue about it.